### PR TITLE
Add minutes

### PR DESCRIPTION
**Pull request by teresanunez**

### CODE REVIEW

**Review by reszelaz — APPROVED** [review]

File: 20200821/MINUTES.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+  David is working in two new controllers talking directly to the
+  hardware. Some problems trying to understand the behaviour of the
+  TriggerGateController, it does not work with the time scan.
+
+Solaris:
+- issues about Sardana customization. It was asked for an easy way to set
+  the logging level (they want to set it to INFO) and about naming templates
+  (the default ones do not follow their naming convention and renaming by hand
+  is a lot of work). Zibi suggested to use the Sardana custom
+  settings. In solaris they use a sardana config file and also define there
```
Comment (cpt-majkel) [inline 20200821/MINUTES.md:45]: I think it was Aureo who mentioned that they use `dsconfig` to easily rename Sardana elements.

File: 20200821/MINUTES.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+  the name convention. 
+- QtSpock: is integrated in a gui but once the scan is finished the command
+  prompt does not appear, they found a way to workaround this but is not
+  practicable for the users.
+- they are now working on making scans depending on the status of the beam.
+  There are many implementations (DESY, MaXIV, Alb), it should be unified
```
Comment (cpt-majkel) [inline 20200821/MINUTES.md]:
```suggestion
  There are many implementations (DESY, MaXIV, Alba), it should be unified
```

File: 20200821/MINUTES.md [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+  (a dedicated meeting will be scheduled for that)
+- how to import modules from controllers is an issue. SEP19 improves
+  that but it is known that it should be improved in sardana. For now the
+  python path environment could be used, or the PoolPath property of the Pool
+  device.
+
+DESY:
+- tests of sardana3 were done: it is already possibleto use at DESY with
```
Comment (cpt-majkel) [inline 20200821/MINUTES.md]:
```suggestion
- tests of sardana3 were done: it is already possible to use at DESY with
```

File: 20200821/MINUTES.md [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+    (issue #1344).
+    Both issues require major changes. They will be investigated.
+  - from David (MaXIV)
+    * asked for timestamps in the scan, there is an issue about that. Perhaps
+    could be configurable to get an extra timestamp for each channel, other
+    question is how to put it in NeXus.
+
+  - from Solaris
+    * plugin interface for storing scan state, considering the case if the
+    scan was crashed and how to resume when starging again, it is not possible
```
Comment (cpt-majkel) [inline 20200821/MINUTES.md]:
```suggestion
    scan was crashed and how to resume when starting again, it is not possible
```

File: 20200821/MINUTES.md [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+          the [docs](https://sardana-controls.org/devel/guide_migration/index.html)
+	  It is done. 
+ 
+    - [ ] Alba propose examples on how to program without `on_stop()` and `on_abort()` Macro methods.
+          MAX IV and DESY need to test and review the new implementation.
+	  Still not proposed by Alba, they will provide it.
+
+    - [ ] MAX IV will provide a docker image to add to CI.
+          It was some idea for extending the continuous integration, but is
+	  still not done. We will have a look to was Taurus did, they use
```
Comment (cpt-majkel) [inline 20200821/MINUTES.md]:
```suggestion
	  still not done. We will have a look to what Taurus did, they use
```

**Review by cpt-majkel — APPROVED** [review]

**Comment by teresanunez** [comment]
Hi Michal, 
many thanks for the corrections, I have just applied them. 

Best Regards, 
Teresa

**Review by aureocarneiro — APPROVED** [review]

**Pull request merged**